### PR TITLE
Shootout updates for Chapel 1.22

### PR DESCRIPTION
**Pull request by bradcray**
This brings our "submitted shootouts" directory back in sync with the CLBG website and turns the tests that were disabled back on again.

Due to the large number of Chapel 2.0-focused breaking changes in Chapel 1.22, we ended up futurizing or otherwise disabling a large number of shootout benchmarks.  This re-enables them and brings them back into sync with the CLBG website.

Specifically, it:

* changes the `clbg-diff.py` script to:
  - reverse the order of the items passed to diff to match my natural intuition
  - adds some spacing and formatting to some prints that glommed together
  - updated to what I believe is the new shootout for grabbing sources (seems to work)
* adds a copy of Engin's bytes-based, naming it regexdna3.chpl, reflecting the website's naming
* renames binarytrees2.chpl to binarytrees3.chpl reflecting the website's naming and updates it
* renames fannkuchredux.chpl to fannkuchredux2.chpl, " " " " " "
* renames fasta4.chpl to fasta5.chpl, " " " " " "
* renames knucleotide2.chpl to knucleotide3.chpl, " " " " " "
* renames mandelbrot2.chpl to mandelbrot3.chpl, " " " " " "
* renames nbody.chpl to nbody2.chpl, " " " " " "
* renames pidigits.chpl to pidigits3.chpl, " " " " " "
* renames regexredux.chpl to regexredux2.chpl, " " " " " "
* renames revcomp.chpl to revcomp2.chpl, " " " " " "
* updates fasta3.chpl to make it work again
* updates mandelbrot.chpl to make it work again

### CODE REVIEW

**Comment by bradcray** [comment]
@ronawho: Would you review once you get back?  This is mostly rote/mechanical, with the exception of some of the perfgraphs stuff for cases that were relying on the default filename-based label, which I put in comments to preserve (I believe correctly... but may very well have gotten wrong)

File: test/studies/shootout/submitted/regexdna-submitted.graph [hunk 1]
```diff
@@ -1,5 +1,5 @@
-perfkeys: real
-graphkeys: release
-files: regexdna-release.dat
+perfkeys: real real real
```
Comment (ronawho) [inline test/studies/shootout/submitted/regexdna-submitted.graph:1]: Should be comma separated

**Comment by ronawho** [comment]
Other than the .graph file comment, looks good. I ran ` start_test --performance test/studies/shootout/` before and after this PR to check if any .dat file got corrupted or whatever and no errors were reported.

File: test/studies/shootout/submitted/regexdna-submitted.graph (hunk 1 above)
Comment (bradcray) [reply test/studies/shootout/submitted/regexdna-submitted.graph:1]: That's so weird that I missed that... While typing it, I was thinking "it's so dumb that these aren't comma-separated.  I'm not sure where I got it into my head that they weren't.  Not from any of the other files in this directory, apparently.  But also weird: in testing, I didn't notice anything barking at me that it was wrong (though maybe it just didn't bark loudly enough for me to notice... I was cooking dinner at the time).

**Pull request merged**